src/frontend/src/components/notice/NoticeList.vue

```diff
@@ -41,11 +41,36 @@
 import { mapGetters } from "vuex";
 
 export default {
+  methods: {
+    getNotices() {
+      const param = {
+        noticeType: this.fetchedNoticeType,
+        jobPosition: this.fetchedJobPosition,
+        language: this.fetchedLanguage
+      };
+      const queryParam = new URLSearchParams(param).toString();
+      this.$store.dispatch("FETCH_NOTICES", queryParam);
+    }
+  },
   computed: {
+    ...mapGetters(["fetchedNoticeType"]),
+    ...mapGetters(["fetchedJobPosition"]),
+    ...mapGetters(["fetchedLanguage"]),
```

Comment:
```suggestion
    ...mapGetters(["fetchedNoticeType", "fetchedJobPosition", "fetchedLanguage"])
```
처럼 하나의 mapGetters를 사용할 수도 있을 것 같네요!